server/sonar-server/src/main/java/org/sonar/server/platform/web/WebPagesFilter.java

```diff
@@ -47,15 +52,24 @@
 
   private static final String CACHE_CONTROL_HEADER = "Cache-Control";
   private static final String CACHE_CONTROL_VALUE = "no-cache, no-store, must-revalidate";
-
-  private static final String CONTEXT_PLACEHOLDER = "%WEB_CONTEXT%";
+  private static final String WEB_CONTEXT_PLACEHOLDER = "%WEB_CONTEXT%";
+  private static final String DEFAULT_HTML_PATH = "/index.html";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor your code to get this URI from a customizable parameter. ([squid:S1075](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1075))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3124&issues=AWIaWOQJ94KwkppSCsI8&open=AWIaWOQJ94KwkppSCsI8)

[ISSUE_KEY]: AWIaWOQJ94KwkppSCsI8